main-app/app/scripts/controllers.js

```diff
@@ -1,28 +1,43 @@
-noughtsAndCrossesApp.controller('noughtsAndCrossesController',function ($scope,gameModel,gameApi){
+noughtsAndCrossesApp.controller('noughtsAndCrossesController',function ($scope,$http,$q,gameModel,gameApi){
```

Comment:
You don't use the $http service in the controller anyhwere (which is right - the game api proxy wrappers this up), so don't inject it.